app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def wrapper(*args) -> Any:
+        if args not in cache_dict:
+            cache_dict.update({args: func(*args)})
```

Comment:
```suggestion
            cache_dict[args] = func(*args)
```
No need to use update

Reply:
didn't think about it. Thank you 
Fixed it

Reply:
Didn't think about it. Thank you
Fixed it